OpenRA.Mods.Common/Traits/TooltipDescription.cs

```diff
@@ -16,6 +16,7 @@ namespace OpenRA.Mods.Common.Traits
 	[Desc("Additional info shown in the battlefield tooltip.")]
 	public class TooltipDescriptionInfo : ConditionalTraitInfo
 	{
+		[TranslationReference]
 		[Desc("Text shown in tooltip.")]
 		public readonly string Description = "";
```

Comment:
```suggestion
		public readonly string Description;
```

For better linting. This property is the whole point of the trait so it should be required

Reply:
fixed